ia/src/models.py

```
@@ -0,0 +1,33 @@
+class iaParser:
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)